app/core/components/ProposalCard/index.tsx

```diff
@@ -29,15 +30,20 @@ interface IProps {
 }
 
 export const ProposalCard = (props: IProps) => {
+  const navigate = useNavigate();
   const stopEvent = (event: React.MouseEvent<HTMLElement>) =>
     event.stopPropagation();
 
   const prefersDarkMode = useMediaQuery("(prefers-color-scheme: dark)");
 
+  const handleCardClick = () => {
+    navigate(`/projects/${props.id}`);
+  };
+
   return (
     <>
       <Card>
-        <CardActionArea sx={{ height: "100%" }} href={`/projects/${props.id}`}>
+        <CardActionArea sx={{ height: "100%" }} onClick={handleCardClick}>
```

Comment:
This change is because with href it renders the entire CardActionArea as an <a> and then inside it we have other <a> tags for the Tier links. So then we are rendering a tags inside other a tag which is not compliant and that is why we are getting errors on the browser console

Reply:
Then I would rather we make only the title, avatar image clickable with regular links, than lost the cmd+click functionality. But that is my opinion. Don't know what the rest think.